stdlib/3.4/asyncio/tasks.pyi

```diff
@@ -19,10 +24,24 @@ def as_completed(fs: Sequence[_FutureT[_T]], *, loop: AbstractEventLoop = ...,
 def ensure_future(coro_or_future: _FutureT[_T],
                   *, loop: AbstractEventLoop = ...) -> Future[_T]: ...
 async = ensure_future
-# TODO: gather() should use variadic type vars instead of _TAny.
-_TAny = Any
-def gather(*coros_or_futures: _FutureT[_TAny],
-           loop: AbstractEventLoop = ..., return_exceptions: bool = False) -> Future[List[_TAny]]: ...
+@overload
+def gather(coro_or_future1: _FutureT[_T1],
+           loop: AbstractEventLoop = ..., return_exceptions: bool = False) -> Future[Tuple[_T1]]: ...
```

Comment:
Since this is inspired by `zip` and `zip` does not actually return a tuple either (it returns a `zip` object), I feel slightly better about lying.